### PR TITLE
Temple vault: Sunken Temple

### PR DESCRIPTION
**Pull request by alexjurkiewicz**
An ancient temple, mostly destroyed and covered in water. 6-12 altars,
with the missing altars replaced by faded ones.

### CODE REVIEW

**Comment by alexjurkiewicz** [comment]
dont merge me, might have better versions coming

**Comment by alexjurkiewicz** [comment]
temple-crushed

**Comment by alexjurkiewicz** [comment]
screen shot 2015-11-16 at 5 24 13 pm

File: crawl-ref/source/dat/des/portals/ossuary.des [hunk 1]
```diff
@@ -284,6 +284,20 @@ Gcc'ccG
    @
 ENDMAP
 
+NAME: chequers_ossuary_entry_cave
+TAGS: transparent
+MAP
```
Comment (wheals) [inline crawl-ref/source/dat/des/portals/ossuary.des]: add `: ossuary_portal(_G)` above the `MAP` to get it to work right

**Comment by alexjurkiewicz** [comment]
Thanks @wheals -- should be right now

**Pull request merged**